src/vivarium_gates_child_iv_iron/utilities.py

```diff
@@ -128,6 +128,29 @@ def _get_standard_deviation(
     return sd
 
 
+def get_norm_from_quantiles(mean: float, lower: float, upper: float,
```

Comment:
Don't use these functions. You want to use the existing functions above `get_norm` and `get_truncnorm`.